packages/synapse-react-client/src/components/GenericCard/GenericCard.tsx

```diff
@@ -617,6 +636,22 @@ class _GenericCard extends Component<GenericCardPropsInternal> {
               </Box>
             )}
           </div>
+          {includeCitation && (
+            <Box
+              sx={{
+                display: 'flex',
```

Comment:
It doesn't seem like flex properties are being used, but you can keep this if I'm wrong
```suggestion
```